Makefile

```diff
@@ -70,13 +66,13 @@ $(GOBIN)/auth0-cli-config-generator:
 	go install ./pkg/auth0-cli-config-generator
 
 run-integration:
-	auth0-cli-config-generator && commander test commander.yaml
+	auth0-cli-config-generator && commander test commander.yaml --verbose
 .PHONY: run-integration
 
 # Delete all test apps created during integration testing
 integration-cleanup:
 	./integration/test-cleanup.sh
 .PHONY: integration-cleanup
 
-integration: $(GOBIN)/auth0-cli-config-generator $(GOBIN)/commander run-integration integration-cleanup
+integration: build $(GOBIN)/auth0-cli-config-generator $(GOBIN)/commander run-integration integration-cleanup
```

Comment:
@as-herzog I kept the `integration-cleanup` when I solved merge conflicts. But I think that is swallowing potential error reports from `commander` when tests failed.